tensorboard/data/provider.py

```diff
@@ -652,10 +655,17 @@ class HyperparameterSessionRun:
 
 @dataclasses.dataclass(frozen=True)
 class HyperparameterSessionGroup:
-    """A group of runs logically executed together with the same hparam values.
+    """A group of sessions logically executed together with the same hparam values.
 
-    The group of runs may have, for example, combined to generate and test a
-    single model or other artifacts.
+    A session generally represents a particular execution of a job with a given
+    set of hyperparameter values. A session may contain multiple related runs
+    executed together to train and validate a model.
+
+    Often a session group will contain only a single session. However, in some
+    scenarios, the same hyperparameters will be used to execute multiple jobs
+    with the idea to aggregate the metrics across those jobs and analyze
+    non-deterministic factors. In that case, a session group will contain multiple
+    sessions.
 
     We assume these groups of runs were executed with the same set of
```

Comment:
Maybe we should just delete this paragraph. I read it now and don't think it provides a lot of value.

Reply:
Removed, assuming you are referring to the second added paragraph.

Reply:
No I was talking about the paragraph that starts with "We assume these groups..."

Reply:
oops, fixed!